nixos/modules/services/misc/dendrite.nix

```diff
@@ -222,6 +222,13 @@ in
         for available options with which to populate settings.
       '';
     };
+    openRegistration = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = ''
+        Really enable open registration.
```

Comment:
Maybe I just lack sufficient context to understand, but what does this option does? Could this description be improved?